public/js/util/create-store.js

```diff
@@ -50,5 +67,13 @@ module.exports = (opts = {}) => {
     window.devToolsExtension() :
     f => f;
 
-  return applyMiddleware(...middleware)(devtools(createStore));
+  return compose(
+    applyMiddleware(...middleware),
+    DevTools.instrument()
```

Comment:
You need to flag this off, because this is where all the perf problems are. Check if the `reduxDevtools` flag is enabled here and only instrument it if so.

I'd say that you should only export `DevTools` if it's enabled also, and the in the `App` render function just do `DevTools ? DevTools() : null` and avoid the `DevToolsPanel` function.